src/sql/parts/modelComponents/modelViewContent.component.ts

```diff
@@ -71,7 +71,7 @@ export class ModelViewContent extends ViewBase implements OnInit, IModelView {
 
 	@memoize
 	public get connection(): sqlops.connection.Connection {
-		if (!this._commonService.connectionManagementService) {
+		if (!this._commonService.connectionManagementService || !this._commonService.connectionManagementService.connectionInfo) {
```

Comment:
this looks like it should fix the nullref and we should take this change.  But in the case of #1724 I don't think the `connectInfo` is supposed to be undefined since the dialog is being displayed in the context of the dashboard and should have a valid connection context.

Am I missing something?

Reply:
Our current system (connection management service) requires having a uri to have a connection, since the dialogs aren't launched with a uri (at least thats what it looks like to me) there's no way for them to have a connection.

Reply:
Agreed, we will want to add a way to give a URI to dialogs when launching them, but don’t have that right now. 

We’ll take this fix too because some dialogs won’t have connection URIs and this bug would break those ones even once URI logic gets added